app/assets/stylesheets/govuk-component/_related-items.scss

```diff
@@ -0,0 +1,32 @@
+.govuk-related-items {
+  border-top: 10px solid $mainstream-brand;
+  padding-top: 5px;
+
+  h2 {
+    @include core-24;
+    font-weight: 600;
+    color: $text-colour;
+    margin-top: 0.3em;
+    margin-bottom: 0.5em;
+  }
+
+  ul {
+    @include core-16;
+    list-style: none;
+    margin: 0 0 1.25em 0;
+    padding: 0;
+    overflow: hidden;
```

Comment:
I'm not sure `overflow: hidden` is needed here (as the child elements aren't floated), but if it is being used a clearfix, consider using the `contain-floats` [placeholder from the front end toolkit](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_shims.scss#L45).